feature/complete/src/main/kotlin/com/nexters/bandalart/android/feature/complete/CompleteScreen.kt

```diff
@@ -32,31 +58,132 @@ internal fun CompleteScreen(
   modifier: Modifier = Modifier,
 ) {
   Surface(
-    modifier = modifier
-      .fillMaxSize()
-      .padding(16.dp),
+    modifier = modifier.fillMaxSize(),
   ) {
-    Column(
-      modifier = Modifier.fillMaxSize(),
-    ) {
-      Text(
-        text = "Complete Screen",
-        textAlign = TextAlign.Center,
-        fontSize = 16.sp,
-      )
-      Spacer(
-        modifier = Modifier.height(16.dp),
-      )
+    Box {
       Column(
-        modifier = Modifier.wrapContentWidth(),
-        verticalArrangement = Arrangement.spacedBy(16.dp),
+        Modifier.fillMaxSize(),
         horizontalAlignment = Alignment.CenterHorizontally,
       ) {
-        NavButton(
-          onClick = onNavigateBack,
-          route = "Home",
+        Box(
+          modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
+        ) {
+          Row {
+            val image = painterResource(id = R.drawable.ic_right_arrow)
```

Comment:
```suggestion
Icon(
  modifier.clickable(onClick = onNavigateBack),
  imageVector = Icons.Default.ArrowForwardIos,
  contentDescription = "Right Arrow Icon",
)
```
            
나는 Default.~~로 개발해서 통일할 필요가 있다고 보는데, 딱히 디자이너한테 요구할 아이콘이 없으면 이 방법은 어때?! Icon 내부에서 tint도 가능하다구..!?

Reply:
나도 고민한 부분이었는데 한번 적용해보고 별로 다르지 않으면 기본 Default 로 해도 될지도

Reply:
보구 판단하자구 보구~

Reply:
Default 로 적용해도 티가 안날정도로 유사하면 Default 를 사용하는걸로 하자 내가 적용한 것들 중엔 다른 부분들이 몇몇 보이네ㅠ